css/styles.scss

```diff
@@ -1,9 +1,100 @@
 @import "./variables/colors";
 @import "./variables/variables";
+@import "./variables/mixins";
 
 html,
 body {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  background-color: $grey;
+  font-family: "Titillium Web";
+}
+.container {
+  width: 30vw;
+  height: 95vh;
+  margin-top: 5%;
+  background-color: $white;
+  border-radius: 15px 15px 0 0;
+  .header {
+    @include flex($wrap: wrap, $align-content: space-between);
```

Comment:
when applying mixin with include, you don't do flex($direc: column), just flex(column)